tests/test_c_parser.py

```diff
@@ -618,6 +623,12 @@ def test_sizeof(self):
                         ['TypeDecl',
                             ['IdentifierType', ['int']]]]]])
 
+    def test_alignof(self):
```

Comment:
I'd like to see more parser tests here.

At least one test separately for `_Alignof` without `_Alignas` (look at the existing `sizeof` tests)
A test for `_Alignas` with a true constant like `4`
A test for `_Alignas` with a type inside e.g. `_Alignas(int)`.

Reply:
Done, thanks.